shared_model/validators/field_validator.cpp

```diff
@@ -416,24 +416,16 @@ namespace shared_model {
       for (size_t ix = 0; ix < count; ++ix) {
         OrderingEntry const& entry = ptr[ix];
 
-        if (entry.field >= Field::kMaxValueCount) {
-          return ValidationError("Ordering",
-            {
-              fmt::format("Passed field value is {} , while it must be between [0, {}) "
-                , (size_t)entry.field
-                , (size_t)Field::kMaxValueCount
-              )
+        if (entry.field >= Field::kUnknownValue) {
```

Comment:
```suggestion
        if (entry.field >= Field::kMaxValue) {
```
when we compare using `>=`, using `kMaxValue` seems more reasonable